<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EO9AL9V" failures="1" tests="16" name="Test" time="333.371" errors="0" timestamp="2025-01-06T18:12:23 IST">
  <testcase classname="com.test.Login" name="RegisterInvalidEmail" time="3.211"/>
  <testcase classname="com.test.Login" name="RegisterPasswordEmpty" time="1.555"/>
  <testcase classname="com.test.Login" name="RegisterSuccess" time="41.309">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.Login.RegisterSuccess(Login.java:124)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- RegisterSuccess -->
  <testcase classname="com.test.Login" name="LoginInvalidEmail" time="2.25"/>
  <testcase classname="com.test.Login" name="LoginInvalidcredentials" time="41.439"/>
  <testcase classname="com.test.Login" name="LoginSuccess" time="5.661"/>
  <testcase classname="com.test.Login" name="BotEmptyFieldValidation" time="9.325"/>
  <testcase classname="com.test.Login" name="BotCreateSuccess" time="42.353"/>
  <testcase classname="com.test.Login" name="BotUpdateEmptyValidation" time="7.416"/>
  <testcase classname="com.test.Login" name="BotUpdateSuccess" time="42.817"/>
  <testcase classname="com.test.Login" name="Botdelete" time="3.743"/>
  <testcase classname="com.test.Botlist" name="LoginSuccess" time="6.079"/>
  <testcase classname="com.test.Botlist" name="BotCreateSuccess" time="30.32"/>
  <testcase classname="com.test.Botlist" name="txtfilesync" time="28.15"/>
  <testcase classname="com.test.Botlist" name="txtcontentfilesync" time="17.591"/>
  <testcase classname="com.test.Botlist" name="chatbotspreview" time="12.534"/>
</testsuite> <!-- Test -->
